Gemfile

```diff
@@ -3,9 +3,9 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.6'
 
+gem 'hotwire-rails', '~> 0.1.0'
```

Comment:
can we have it below rails? I think that if you leave an empty line like this:
```
gem 'rails', '~> 6.1.0'

gem 'hotwire-rails', '~> 0.1.0'
```

rubocop won't complain. Or you can even add a comment like
```
# Use Hotwire to make the app feel like an SPA
```